<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditor</class>
 <widget class="QMainWindow" name="ScriptEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTScript Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="connectionLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IP исполнителя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="executorIP">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Порт:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="executorPort">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>12346</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="scriptEditor">
      <property name="font">
       <font>
        <family>Courier</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Введите или загрузите QTScript здесь...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="executeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Выполнить скрипт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
